_test/test/goldens/generated_build_script.dart

```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: directives_ordering
```

Comment:
I could have bumped the min version of code builder used in build_runner – then enabled the order option. But that seems silly to push down to our users now...